test/StructureMap.Microsoft.DependencyInjection.Tests/StructureMapContainerTests.cs

```diff
@@ -59,6 +59,14 @@ public void ConfigureAndRegisterDoNotPreventPopulate()
             Assert.NotNull(container.GetInstance<IFakeService>());
             Assert.NotNull(container.GetInstance<IFakeSingletonService>());
             Assert.NotNull(container.GetInstance<IFakeScopedService>());
+
+            var spis = container.GetInstance<IServiceProviderIsService>();
```

Comment:
Oh nevermind, the tests are here.

Would be good to add one for the IEnumerable assertion.

Reply:
There's a test for `IEnumerable` in the inherited `DependencyInjectionSpecificationTests`. That's how I knew to add it. 😁

Reply:
But I guess since we have this (redundant) test here we might as well add the corresponding `IEnumerable<>` checks.